operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java

```
@@ -841,12 +841,22 @@ private V1Probe createReadinessProbe(TuningParameters.PodTuning tuning) {
         .initialDelaySeconds(getReadinessProbeInitialDelaySeconds(tuning))
         .timeoutSeconds(getReadinessProbeTimeoutSeconds(tuning))
         .periodSeconds(getReadinessProbePeriodSeconds(tuning))
-        .failureThreshold(FAILURE_THRESHOLD)
-        .httpGet(
-            httpGetAction(
-                READINESS_PATH,
-                getLocalAdminProtocolChannelPort(),
-                isLocalAdminProtocolChannelSecure()));
+        .failureThreshold(FAILURE_THRESHOLD);
+    try {
+      boolean istioEnabled = Boolean.valueOf(System.getenv("ISTIO_ENABLED"));
+      if (istioEnabled) {
+        readinessProbe = readinessProbe.httpGet(httpGetAction(READINESS_PATH, 8888, false));
+      } else {
+        readinessProbe =
+            readinessProbe.httpGet(
+                httpGetAction(
+                    READINESS_PATH,
+                    getLocalAdminProtocolChannelPort(),
+                    isLocalAdminProtocolChannelSecure()));
+      }
+    } catch (Exception e) {
+      // do nothing
```

Why swallow the exception?